kontonummer.js

```diff
@@ -203,6 +203,15 @@
             account  : 7,
             control  : 10
         }
+    },{
+        name    : 'MedMera Bank',
+        regex   : /^(965[0-9])/,
+        modulo  : 11,
+        lengths : {
+            clearing : 4,
+            account  : 7,
+            control  : 10
```

Comment:
Same here with control number. Comment 2 with full clearing + account number as control

Reply:
You are totally correct. I got confused by what `control` means in this context.

Reply:
I've fixed this in the fixup commit 2bb6cd2b5d745511c2469864d4e5f03634ef0971